src/redux/configureStore.js

```diff
@@ -7,6 +8,6 @@ const rootReducer = combineReducers({
   categories: categoriesReducer,
 });
 
-const store = createStore(rootReducer);
+const store = createStore(rootReducer, applyMiddleware(thunk));
```

Comment:
[OPTIONAL] I would recommend using some handy Redux middleware, like a redux logger. It will help you a lot in the development process. To install it, you need to run:

  ```
  npm i --save redux-logger
  ```

Reply:
Thanks for recommendation